tools/testbench/common_test.c

```diff
@@ -19,7 +19,7 @@
 #include <sof/lib/wait.h>
 #include <sof/audio/pipeline.h>
 #include "testbench/common_test.h"
-#include "testbench/topology.h"
+#include <topology.h>
```

Comment:
I'd move `topology.h` from `${parser_install_dir}/include` to f.e. `${parser_install_dir}/include/tplg_parser`, so it'd be included like `#include <tplg_parser/topology.h>`, just like for sof includes.